packages/block-library/src/post-date/edit.js

```diff
@@ -155,7 +138,114 @@ export default function PostDateEdit( {
 					/>
 				</PanelBody>
 			</InspectorControls>
+
 			<div { ...blockProps }>{ postDate }</div>
 		</>
 	);
 }
+
+function is12HourFormat( format ) {
```

Comment:
Is it worth adding a unit test for this function, since it's a small refactor?

Reply:
Sure is 👍 5f2851c8826c54945861a53b1f459af53b95a8f0